packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -131,16 +132,20 @@ class Checkbox extends StatefulWidget {
   ///
   /// Defaults to [ThemeData.toggleableActiveColor].
   ///
-  /// If [activeColor] is a [MaterialStateColor], it will be resolved in the
-  /// following states:
+  /// If [fillColor] is non-null, this will be ignored.
+  final Color? activeColor;
+
+  /// The color that fills the checkbox when it is checked, in all
+  /// [MaterialState]s.
+  ///
+  /// If this is provided, it will be used over [activeColor].
```

Comment:
And hoverColor, focusColor, right?

Reply:
No, because `hoverColor` and `focusColor` describe the overlay color, `fillColor` is just for the painted checkbox itself.